backend/app.py

```diff
@@ -2071,6 +2078,107 @@ def process_financial_documents():
             "status": "error",
             "message": str(e)
         }), 500
-    
+
+# main.py
+from app_config import IMAGE_PATH
+from summarization import DocumentSummarizer
+
+@app.route('/api/SECEdgar/summarize', methods=['GET'])
```

Comment:
@namtran6701 we might consider using POST since the endpoint seems to process input data and create a summary. A GET request typically doesn't have a body, and here we’re relying on request.get_json()
this is useful resource that might help with naming endpoints:
https://restfulapi.net/resource-naming/